src/cmd-buildextend-live

```diff
@@ -337,11 +337,12 @@ def generate_iso():
             newbuf = buf.replace('@@KERNEL-ARGS@@', kargs)
             # if we injected kargs, also check for an embed area
             if buf != newbuf:
+                karg_area_start = re.search(r'@@KERNEL-ARGS@@', buf)
```

Comment:
One thing I think we want to support natively is "resetting" the kargs to the defaults. So e.g. you can do whatever additions/deletions you want with `coreos-installer iso kargs`, but doing `coreos-installer iso kargs reset` should yield you back the original ISO (with matching checksum).

The way I imagined this working was to just to have another file in the ISO containing the default kargs, and another pointer in the System Area which points to that file. Then `coreos-installer` would know to read the defaults from there when resetting the kargs. WDYT?

Reply:
hmm, i expect user to be responsible for any actions he may do with `kargs & iso`. On the other hand - adding and removing of particular `kargs` is safe - image after such kind of modifications will be same as original.

Reply:
This would require users to try to remember what the original kargs were. Also, in a support context, telling users to `coreos-installer iso kargs reset` to make sure the ISO matches the original is very useful. It also mirrors the functionality we provide for Ignition configs, where `coreos-installer iso ignition remove` allows you to undo easily.

Reply:
This relates to https://github.com/ostreedev/ostree/pull/2217 - if we had that we could offer "factory reset" from the ostree PoV.

Reply:
Right, we want to make sure there's a straightforward way to return the ISO to an unmodified state.

Reply:
> This relates to [ostreedev/ostree#2217](https://github.com/ostreedev/ostree/pull/2217) - if we had that we could offer "factory reset" from the ostree PoV.

as long as it's not merged yet, i've added read-only `.cmdline` to ISO root, so now `reset` works